modules/install-sno-about-installing-single-node-openshift.adoc

```diff
@@ -0,0 +1,8 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="install-sno-about-installing-single-node-openshift_{context}"]
+= About installing Single Node OpenShift (SNO)
+
+Single Node OpenShift (SNO) is a specialized installation. The primary use case is for edge computing workloads, including intermittent connectivity, portable clouds, and 5G radio access networks (RAN) close to a base station. An installation on a single node supports autonomous workloads. The major trade off with an installation on a single node is the lack of high availability.
```

Comment:
here https://github.com/openshift/openshift-docs/pull/38068/files#diff-1568f3f81b2f1a323d90d2b334a89844650334ecd4718f498de1a1cc87472101R251 u dont use capital letters. Lets be consistent

Reply:
Consistent with what word or phrase?

Reply:
single node or Single Node

Reply:
"Single Node OpenShift (SNO)" is a proper noun and should take capitalization, but a "single node" is just a regular noun and doesn't take capitalization. 

/2 cents